scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+function reportFailure() {
+  if [ -n "${SLACK_WEBHOOK}" ] && [ -n "${BUILD_LABEL}" ]
+  then
+    BRANCH_NAME=$(git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/\1/')
+    SLACK_MESSAGE=":failed_build: ${BUILD_LABEL} ${BROWSER} ${BVER} on branch ${BRANCH_NAME}"
```

Comment:
Can we include the `CIRCLE_BUILD_URL` environment variable here? It's a unique URL that we can just visit and will lead us directly to the failing job.